scripts/perf_cmux_adapter.py

```
@@ -0,0 +1,1602 @@
+                }
+                metadata = self._profile_metadata(
+                    phase="churn", work_units=work_units
+                )
+                profile_pool = ThreadPoolExecutor(
+                    max_workers=max(1, len(metadata))
+                )
+                profile_futures = {
+                    profile_pool.submit(self._run_profile, item): item
+                    for item in metadata
+                }
+            except BaseException as error:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `BaseException`

(BLE001)